styles/overrides/_styleguidist.scss

```diff
@@ -3,56 +3,250 @@
 //
 // 1. To override styleguidist styles
 
-.rsg-components__search-header input[type="search"] {
-    width: 100%;
-    padding: $small-space;
-    border: 1px solid $border-color;
+$rsgSearchHeaderWidth: 250px;
 
-    border-radius: 0;
-    background-color: $white;
-    outline: none;
 
-    transition: all .15s ease-in-out;
+// Global Styleguide Styles
+// ---
 
-    &:focus {
-        box-shadow: $small-box-shadow;
+.rsg-style-guide {
+    display: flex;
+    flex: 1 1 auto;
+}
+
+@include for-phone-only {
+    .rsg-style-guide--is-top-level {
+        .rsg-components__search-nav {
+            display: block;
+            width: 100vw;
+
+            a {
+                display: inline-block;
+                padding: $small-space 2px;
+            }
+        }
+
+        .rsg-components__component-section {
+            display: none;
+        }
+
+        [class*="rsg--isChild-"] {
+            display: block;
+            margin: $small-space 0;
+        }
+    }
+
+    // Component view's first header wrapper
+    [id*="-container"] {
+        [class*="rsg--wrapper-"] {
+            display: flex;
+            flex-direction: column-reverse;
+            align-items: flex-start;
+        }
```

Comment:
I can't find where these styles are applied in the DOM 🤔

Reply:
This line moves the "Show All Components" button (the thing that looks like some sort of crosshairs) to live above the component heading as the "<- Back" link on Mobile.

| Desktop | Mobile |
| --- | --- |
| ![screen shot 2018-02-05 at 2 47 39 pm](https://user-images.githubusercontent.com/734535/35832894-cdf7f7ac-0a83-11e8-8d29-ed2ab8c3d2fc.png) | ![screen shot 2018-02-05 at 2 47 49 pm](https://user-images.githubusercontent.com/734535/35832899-d2cb9298-0a83-11e8-9238-6af6e0f23f1d.png) |